### DISCUSSION TITLE
Android with gamepad: How to use spell hotkeys?

### DISCUSSION THREAD

**Discussion by ericmchan** [diasurgical/DevilutionX#6069] [Q&A] [answered]
I'm playing the Android version on Retroid Pocket 3, and can't figure out how to map the spells to hotkeys. When I press and hold Select, I see the four empty squares. But can't figure out how to assign anything to them. I tried various button combos along with touchscreen. Please help! 😅

**Comment by AJenbo** [comment, marked as answer]
press the spellicon, move yo the spell, press the combo

**Comment by ericmchan** [comment]
Thank you! Would never have guessed that in a million years. Appreciate all the hard work you do on this project!

**Comment by echamings** [comment]
Anyone know how to set scrolls on the same hot keys?

**Reply by AJenbo** [reply]
Same, though you can only have one spell bound
